### PR TITLE
#167132923 Setup boilerplate for the frontend

### PR DESCRIPTION
**Pull request by kelvinndmo**
## Description ##
This pull request helps set up the boilerplate folder for the project. I have the following changes proposed:
 - Inside the app folder, we have a `components` folder to hold all our components.
 - I also propose the same for other folders such as services. i.e `services/login.services.ts`

Here is a screenshot of the suggested folder structure:
Screen Shot 2019-07-09 at 4 12 19 PM

## Type of change ##
Please select the relevant option
- [ ] Bug fix(a non-breaking change which fixes an issue)
- [x] New feature(a non-breaking change which adds functionality)
- [ ] Breaking change(fix of feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes
- [ ] End to end test
- [ ] Integration test
- [ ] unit test

## Checklist: ##
- [ ] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes

## Related Pivotal Tracker stories ##
[#167132923](https://www.pivotaltracker.com/story/show/167132923)

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            "main": "src/main.ts",
+            "polyfills": "src/polyfills.ts",
+            "tsConfig": "tsconfig.app.json",
+            "aot": false,
+            "assets": [
+              "src/favicon.ico",
+              "src/assets"
+            ],
+            "styles": [
+              "src/styles.css"
```
Comment (MandelaK) [inline angular.json]: Could you please configure bootstrap?

**Review by akram256 — COMMENTED** [review]
Hey @kelvinndmo , Can you please edit the screenshot in the PR description. Thanks

**Review by joelethan — APPROVED** [review]

**Review by bl4ck4ndbr0wn — APPROVED** [review]

**Comment by malep2007** [comment]
I think this is good to get started

**Review by MandelaK — APPROVED** [review]

**Review by akram256 — APPROVED** [review]
LGTM

**Comment by malep2007** [comment]
@kelvinndmo, please rebase so this can be merged

**Review by akram256 — COMMENTED** [review]
good job

File: angular.json [hunk 2]
```diff
@@ -17,18 +21,29 @@
         "build": {
           "builder": "@angular-devkit/build-angular:browser",
           "options": {
+<<<<<<< HEAD
             "outputPath": "dist/landville-frontend",
+=======
+            "outputPath": "dist/landvile-frontend",
+>>>>>>> 167132923-ch(project-boilerplate): Setup boilerplate for the frontend
```
Comment (joelethan) [inline angular.json]: @kelvinndmo, You pushed changes halfway through the rebase process, could you please rectify this.

**Review by joelethan — DISMISSED** [review]

**Comment by Victoradukwu** [comment]
@kelvinndmo Please work on fixing the CircleCI failure.

File: angular.json (hunk 2 above)
Comment (kelvinndmo) [reply angular.json]: damn sure.On it

**Review by BrianSerem — APPROVED** [review]

**Pull request merged**